includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -95,20 +91,7 @@ public function admin_header() {
 		header( 'Cross-Origin-Opener-Policy: same-origin' );
 		header( 'Cross-Origin-Embedder-Policy: require-corp' );
 
-		ob_start();
-	}
-
-	/**
-	 * Do string replacement and output.
-	 *
-	 * @since 1.6.0
-	 *
-	 * @return void
-	 */
-	public function admin_footer() {
-		$html = (string) ob_get_clean();
-
-		echo $this->replace_in_dom( $html ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Don't we need to end the output buffer?

Reply:
Not necessarily. If outbut buffering is still active when the script ends, PHP outputs the contents automatically. And our callback will be called then.